src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1095,6 +1096,21 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		}
 	}
 
+	focusAccessibilityElement(): void {
+		if (!this.xterm?.raw.element) {
+			return;
+		}
+		this._accessibilityElement = this.xterm.raw.element.querySelector('.xterm-accessibility-buffer') as HTMLElement || undefined;
+		if (!this._accessibilityElement) {
+			return;
+		}
+		const fontFamily = this._configurationService.getValue(TerminalSettingId.FontFamily) || 'Monospace';
+		const fontSize = this._configurationService.getValue(TerminalSettingId.FontSize);
+		this._accessibilityElement.style.fontFamily = fontFamily ? fontFamily.toString() : '';
+		this._accessibilityElement.style.fontSize = fontSize ? fontSize + 'px' : '';
```

Comment:
This part is handled by xterm.js?

https://github.com/xtermjs/xterm.js/blob/41dc93a008dc7bc4044020cfa483210891770b50/src/browser/AccessibilityManager.ts#L346-L349